docs/_documentations/remote-obtainingingressorroute.md

```diff
@@ -0,0 +1,17 @@
+---
+layout: docs
+title: Obtaining an ingress or route
+description: Obtaining an ingress or route
+keywords: users, projects, Kubernetes, LDAP, user management, access management, login, deployment, pod, security, securing Cloud connection
+duration: 5 minutes
+permalink: remoteobtainingingressorroute
+type: document
+parent: installoncloud
+order: 1
+---
+
+## Obtaining an ingress or route
+
+To obtain an ingress or route...
+
+`export INGRESS_DOMAIN=$(kubectl get services --namespace ingress-nginx -o jsonpath='{.items[*].spec.clusterIP}') sudo ifconfig lo0 alias ${INGRESS_DOMAIN}`
```

Comment:
This will only work if the ingress domain was installed as part of `Step 2` in the install doc. 

We should also mention how to discover the ingress domain on OpenShift and/or other Kubes
 
@markcor11 FYI

Reply:
Noted, have rewritten the doc separating the two procedures, will catch up with Mark on Monday to clarify. Docs have changed considerably so closing as complete but subject to another review.